example/materialBall.js

```diff
@@ -8,9 +8,13 @@ import { RGBELoader } from 'three/examples/jsm/loaders/RGBELoader.js';
 import { MeshoptDecoder } from 'three/examples/jsm/libs/meshopt_decoder.module.js';
 import { GUI } from 'three/examples/jsm/libs/lil-gui.module.min.js';
 
-let renderer, controls, sceneInfo, ptRenderer, camera, fsQuad, materials;
+let renderer, controls, sceneInfo, ptRenderer, activeCamera, fsQuad, materials;
+let perspectiveCamera, orthoCamera;
 let envMap, envMapGenerator, scene;
 let samplesEl;
+
+const orthoWidth = 5;
+
```

Comment:
Last question and this otherwise looks great! What is "orthoWidth" in these examples and why it it different in each one?

Reply:
Sets the size of the frame. I eyeballed it so that the image looks similar to the perspective camera (the equivalent value on the perspective camera is FOV, but we don't extract that to a variable because it isn't needed in the resize() callback, but this is.)